client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -92,4 +99,149 @@ describe('FeedbackGeneratorService', function() {
         "more efficiently?"].join(''));
     });
   });
+
+  describe('_getBuggyOutputTestFeedback', function() {
+    it(['should return the next hint in sequence for buggy outputs, ',
+        'provided the code has been changed'].join(''), function() {
+      var buggyOutputTestDict = {
+        buggyFunction: 'AuxiliaryCode.countNumberOfParentheses',
+        messages: [
+          ["Try running your code on '))((' on paper. ",
+           'Did you expect that result?'].join(''),
+          [
+            'Are you making sure the parentheses are properly balanced? () ',
+            'is balanced, but )( is not.'
+          ].join(''),
+          [
+            "It looks like you're counting the number of parentheses, and if ",
+            "you have the same # of each kind, returning true. That's not ",
+            "quite correct. See if you can figure out why."
+          ].join('')
+        ]
+      };
+      var buggyOutputTest = BuggyOutputTestObjectFactory.create(
+        buggyOutputTestDict);
+      var codeEvalResult = CodeEvalResultObjectFactory.create(
+        'some code', 'same output', [], [true], [], false, 
```

Comment:
Yep, I agree it doesn't trigger (since we're not testing that part of the code in this test) but I think it's a good idea to make these calls reflect reality. Thanks!